modules/infrastructure/main.tf

```
@@ -27,109 +25,62 @@ resource "azurerm_service_plan" "main" {
   sku_name            = "Y1" #required / Plan zużycia
 }
 
-# 3. Storage Account
-resource "azurerm_storage_account" "storage" {
-  name                     = var.storage_account_name # required
-  resource_group_name      = azurerm_resource_group.main.name  # required
-  location                 = azurerm_resource_group.main.location  # required
-  account_tier             = "Standard"  # required
-  account_replication_type = "LRS" # required
+# Moving frontend
+moved {
+ from =  module.infrastructure.azurerm_static_web_app.frontend
+ to = module.static_web_app.azurerm_static_web_app.frontend
```

Mogłoby być tak i nawet tak byłoby lepiej. Korzystając z nazw obiektów możesz podawać pełną nazwę względem root module, ale także względem module.infrastructure, co zrobiłeś w linii "to". Jestem ciekawy co by było gdybyś w root module dodatkowo wywoływał moduł nazwany static_web_app... czuję że mogłby powstać konflikt o ile TF nie ma wbudowanej kolejności/preferencji czy traktować to jako ścieżkę relatywną lub absolutną
```suggestion
 to = azurerm_static_web_app.frontend
```

---

Widzę że w kolejnych przypadkach masz tak samo

---

gotowe 👍

---

Wprowadziłem cię w błąd. Wygląda że sugestia mi się zrobiła na złej linii. Na szczęscie już ogarnąłem jak robić wieloliniowe sugestie